specifics/Dhyana/Dhyana.h

```diff
@@ -1,291 +1,506 @@
-	
-};
-
-}	// namespace_ns
-
-#endif	// _DHYANA_H
+/*----- PROTECTED REGION ID(Dhyana.h) ENABLED START -----*/
+//=============================================================================
+//
+// file :        Dhyana.h
+//
+// description : Include file for the Dhyana class
+//
+// project :     Device specific for Dhyana detector
+//
+// This file is part of Tango device class.
+// 
+// Tango is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// Tango is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with Tango.  If not, see <http://www.gnu.org/licenses/>.
+// 
+// $Author:  $
+//
+// $Revision:  $
+// $Date:  $
+//
+// $HeadURL:  $
+//
+//=============================================================================
+//                This file is generated by POGO
+//        (Program Obviously used to Generate tango Object)
+//=============================================================================
+
+
+#ifndef Dhyana_H
+#define Dhyana_H
+
+#include <tango.h>
+#include "Factory.h"
+#include "lima/HwInterface.h"
+#include "lima/CtControl.h"
+#include "lima/CtAcquisition.h"
+#include "lima/CtImage.h"
+#include <yat4tango/PropertyHelper.h>
+#include <yat4tango/InnerAppender.h>
+#include "DhyanaInterface.h"
+
+
+#include "Factory.h"
+
+#include "lima/HwInterface.h"
+#include "lima/CtControl.h"
+#include "lima/CtAcquisition.h"
+#include "lima/CtImage.h"
+
+#include <yat4tango/PropertyHelper.h>
+#include <yat4tango/InnerAppender.h>
+
+#include "DhyanaInterface.h"
+#include "DhyanaCamera.h"
+
+#define MAX_ATTRIBUTE_STRING_LENGTH     256
+#define TUCAM_VERSION  "1.0.0"
+
+#define CHANNEL_1 0
+#define CHANNEL_2 1
+#define CHANNEL_3 2
+
+using namespace lima;
+using namespace std;
+using namespace yat4tango;
+ 
+/*----- PROTECTED REGION END -----*/	//	Dhyana.h
+
+/**
+ *  Dhyana class description:
+ *    Interface the camera Dhyana using  the TUCAM Library
+ */
+
+namespace Dhyana_ns
+{
+enum _channel1Enum : short
+{
+} ;
+typedef _channel1Enum channel1Enum;
+
+enum _channel2Enum : short
+{
+} ;
+typedef _channel2Enum channel2Enum;
+
+enum _channel3Enum : short
+{
+} ;
+typedef _channel3Enum channel3Enum;
+
+enum _edge1Enum : short
+{
+} ;
+typedef _edge1Enum edge1Enum;
+
+enum _edge2Enum : short
+{
+} ;
+typedef _edge2Enum edge2Enum;
+
+enum _edge3Enum : short
+{
+} ;
+typedef _edge3Enum edge3Enum;
+
+/*----- PROTECTED REGION ID(Dhyana::Additional Class Declarations) ENABLED START -----*/
+
+enum class TriggeroutMode : short
+{
+	EXPOSURESTART,
+	READOUT,
+	GLOBAL

```

Comment:
add a k and use the mixed case notation, eg: kExposureStart 

idem for all Enums